### ISSUE TITLE
https://porno1.hu/

### ISSUE THREAD

**Issue by Beladeep** [dobbelina/repository.dobbelina#1246]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.

### Description

Dear developers

I would like you to add this page to your fantastic addon.

Thank you very much.

### If it's a site addition request, add url below

https://porno1.hu/

### URL to DMCA information

https://porno1.hu/dmca/

**Comment by Beladeep** [comment]
Dear developers

I have tried to create this site myself as a coustom site.
I was able to create the categories and tags folders. I was also able to list the included folders.
Unfortunately I can't list the video titles anymore. Whatever I try, it does not bring any results.
I guess there must be some regex problem. Maybe the origin coustom site I modified from does not match the web structure of https://porno1.hu/

Could someone be so kind and fix the problems?

I'll upload the status I've gotten so far.

[coustom-porno1.hu.zip](https://github.com/dobbelina/repository.dobbelina/files/13801234/coustom-porno1.hu.zip)

Thank you very much, and happy new year everyone!

**Comment by Beladeep** [comment]
Dear @dobbelina

First of all, I would like to thank you for your work.
The site is working well now. :-)
I am amazed at how fast and professional you are.
May I ask how long it takes you to make a site like this?
There is one small part of the code missing that doesn't work properly.
No tags folder. I tried to do it myself, but I only managed to create an empty folder or error... :-(
I'm trying to learn as much as I can, but I still need a lot more. I don't have much help, only chatgpt, but he often writes nonsense.
Thank you very much if you correct even that small deficiency.

![no tags](https://github.com/dobbelina/repository.dobbelina/assets/132296655/bce44def-aa25-43b4-a2a0-2490692c7b9a)

The best version yet:
[coustom-porno1.hu.zip](https://github.com/dobbelina/repository.dobbelina/files/13804337/coustom-porno1.hu.zip)
